docs/Misc/How-to-setup-Torguard-for-port-forwarding.md

```diff
@@ -122,72 +100,33 @@ Then click on `Manage` and select `Port Forward Request`.
 
     ![!Port Forward Request](images/torguard/req_port_fwd.png)
 
-### VPN Tunnel Type
-
-Choose your preferred VPN Tunnel type.
-
-#### OVPN Request Port Forward
-
-??? abstract "OVPN Request Port Forward"
-
-    ![!Request New Port Forward OVPN](images/torguard/request-new-pfw-ovpn.png)
-
-    1. That's the `IP` we found earlier.
-    1. Select `UDP`.
-    1. Select a `Port/Auth` I personally use `995|SHA1`.
-    1. Select the `Protocol` `TCP`.
-    1. I suggest to use a high `Port` number 10.000+ or a game port you don't use, don't use the default torrent ports 6881- 6889
-        (This is also the port you will use in your torrent client).
-    1. Click on the `+` sign and do the same with the `UDP` `Protocol`.
-    1. Then click on `Submit Request`.
-
-    If everything went according to plan you will see something like this.
-
-    ![status](images/torguard/status.png)
-
-    You will also receive a e-mail with the ports you forwarded.
-
-    !!! Note
-
-        Normally it doesn't take more then a few seconds/minutes to get the ports approved if it takes longer or you get `PENDING` I suggest to ping the server again (or choose another sever) and you often get a new IP and retry it again with the new IP.
-
-#### WireGuard Request Port Forward
-
-??? abstract "WireGuard Request Port Forward - [Click to show/hide]"
-
-    ![!Request New Port Forward WireGuard](images/torguard/request-new-pfw-wireguard.png)
-
-    1. That's the `IP` we found earlier.
-    1. Select `UDP`. (:bangbang: **KEEP THIS ON UDP FOR WireGuard** :bangbang:)
-    1. Select a `Port/Auth` and select `WireGuard`.
-    1. Select the `Protocol` `TCP`.
-    1. I suggest to use a high `Port` number 10.000+ or a game port you don't use, don't use the default torrent ports 6881- 6889
-        (This is also the port you will use in your torrent client).
-    1. Click on the `+` sign and do the same with the `UDP` `Protocol`.
-    1. Then click on `Submit Request`.
+### Port Forward Request
 
-    !!! warning "WARNING"
+![!Request New Port Forward WireGuard](images/torguard/request-new-pfw-wireguard.png)
 
-        :bangbang: DON'T CHANGE OPTION 2 `UDP` to `TCP` :bangbang:
+1. That's the `IP` we found earlier.
+1. Select `UDP`. (:bangbang: **KEEP THIS ON UDP FOR WireGuard** :bangbang:)
+1. Select a `Port/Auth` and select `WireGuard`.
+1. Select the `Protocol` `TCP`.
+1. I suggest to use a high `Port` number 10.000+ or a game port you don't use, don't use the default torrent ports 6881- 6889
```

Comment:
```suggestion
1. I suggest to use a high `Port` number 10000+ or a game port you don't use, don't use the default torrent ports 6881- 6889
```
no comma, this is a port number not a numeric number - do not use a separator

Reply:
yep. this.